syntax = "proto3";

service MLService {
    rpc RecommendationService (StudentInfo) returns (CoursesInfo);
}

message StudentInfo {
    int32 student_id = 1;
    int32 semester_filter = 2;
    int32 k = 3;
}

message CoursesInfo {
    string data = 1;
}
